openwillis/measures/text/util/characteristics_util.py

```diff
@@ -111,7 +104,6 @@ def create_index_column(item_data, measures):
     """
     index = 0
     for item in item_data:
```

Comment:
I removed the nested loops and split `create_turns_aws` to be more manageable. But I don't think we should add exception handling there. Since this is the standard preprocessing function before extracting measures from the data, it shouldn't fail ever. If it fails then the whole function should fail.